lit/direct/complex1.thorin.eval

```diff
@@ -0,0 +1,63 @@
+.import core;
+.import debug;
+.import direct;
+.import mem;
+// .lam .extern internal_diff_core_icmp_xYgLE __690955: .Nat → .Cn [«2:.Nat; .Idx __690955», .Cn [(.Idx 2), .Cn [(.Idx 2), .Cn «2:.Nat; .Idx __690955»]]] = {
```

Comment:
what's the deal with this commented code section?

Reply:
It was the remaining externals as the code failed/was aborted before the external sweep pass ran.